### PR TITLE
[docs] Add documentation about CHARRA PoC on PC

### PR DESCRIPTION
**Pull request by 0xDen**
Signed-off-by: Denis Semakin <insane79@mail.ru>

### CODE REVIEW

**Comment by macpijan** [comment]
@0xDen Please link this document in the Table of Contents: https://github.com/fobnail/docs/blob/main/mkdocs.yml#L23

File: docs/charra-concept-on-pc.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+cd charra
+make
+```
+
+3. Run Attester and Verifier on two different consoles:
+![Attester](images/fobnail-run-attester-verifier.png)
+
+4. Got the result.
+![Verifier](images/fobnail-attestation-results.png)
+
```
Comment (macpijan) [inline docs/charra-concept-on-pc.md:33]: No need to leave extra empty lines

Comment (0xDen) [reply docs/charra-concept-on-pc.md:29]: Done

Comment (macpijan) [reply docs/charra-concept-on-pc.md:29]: I'm pretty sure these are still there

**Pull request closed without merging**